.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I have added `frontend/.vscode/` in the base level `.gitignore` and I believe you have also added `.vscode/*` in `frontend/.gitignore` so I'm not sure why this file is being tracked.